Practice9/src/main/java/com/academiaDigital/softwareArchitecture/practice9/provider/Name.java

```diff
@@ -0,0 +1,16 @@
+package com.academiaDigital.softwareArchitecture.practice9.provider;
+
+public class Name implements INameProvider {
+    private String name;
+    private Preferences preferences;
+
+    public Name(String name, Preferences preferences) {
+        this.name = name;
+        this.preferences=preferences;

```

Comment:
why is this needed?

Reply:
You are right, I forgot delete that property (I was thinking use preference but then I forgot erase it).